vignettes/tutorial-create-your-first-rhino-app.Rmd

```diff
@@ -7,11 +7,28 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+<style>
+
+.app-preview {
+
+  margin: 1.5em 0.75em;
+  padding: 0.25em;
+  box-shadow:
+    0 3.9px 4.6px rgba(0, 0, 0, 0.08),
+    0 12.3px 8.4px rgba(0, 0, 0, 0.056),
+    0 18.8px 19.2px rgba(0, 0, 0, 0.037),
+    0 22px 40px rgba(0, 0, 0, 0.019);
```

Comment:
Do we need 4 separate shadows with somewhat funky fractional values? I'm fine with it, but perhaps this could be simplified for a similar visual effect :wink:

Reply:
I am open to changing this. I learned this from a css-tricks article and have been using it for all my projects as a sass mixin. Open to changing this if you want